docs/src/main/asciidoc/sending-emails.adoc

```diff
@@ -75,7 +75,7 @@ The following table lists the properties that can be configured:
 | port | Integer | the SMTP server port | `25`
 | username | String | the username |
 | password | String | the password |
-| ssl | Boolean | Enables or disables SSL | `false`
+| ssl | Boolean | Enables or disables SSL. If enabled, you must also link:native-and-ssl-guide.html[set quarkus.ssl.native to true] when building a native image. | `false`
```

Comment:
It's minor but having backticks around `quarkus.ssl.native` would be nice

Reply:
Good catch. Should be good. :-)